.env.example

```diff
@@ -1,15 +1,25 @@
 HOST=127.0.0.1
 PORT=5000
 
+# uvicorn variable, allow https behind a proxy
+FORWARDED_ALLOW_IPS="*"
+
 DEBUG=false
 
 LNBITS_ALLOWED_USERS=""
 LNBITS_ADMIN_USERS=""
 # Extensions only admin can access
-LNBITS_ADMIN_EXTENSIONS="ngrok"
+LNBITS_ADMIN_EXTENSIONS="ngrok, admin"
```

Comment:
What does ngrok do?

Reply:
its an admin extension that allows the user to set up a reverse proxy, similar to what BTCPayServer does

Reply:
This leaks your IP, right?

Sounds nifty, as long as users are aware